### PR TITLE
[resolves #5] find student request by student id

### PR DESCRIPTION
**Pull request by GabrielaMachado**

### CODE REVIEW

**Review by DanielaDiazP — APPROVED** [review]

**Review by conrado606 — APPROVED** [review]

File: app/controllers/RequestStudentController.js [hunk 1]
```diff
@@ -13,3 +13,18 @@ RequestStudentController.save = async (req, res, next) => {
     return next(error);
   }
 };
+RequestStudentController.findByStudentId = async (req, res, next) => {
+  try {
+    // eslint-disable-next-line camelcase
```
Comment (ingesofteam) [inline app/controllers/RequestStudentController.js]: quitar esto

File: app/controllers/RequestStudentController.js [hunk 2]
```diff
@@ -13,3 +13,18 @@ RequestStudentController.save = async (req, res, next) => {
...
+    const { query: { student_id } } = req;
```
Comment (ingesofteam) [inline app/controllers/RequestStudentController.js]: const { query: { student_id:studentId } } = req;

File: app/controllers/RequestStudentController.js [hunk 3]
```diff
@@ -13,3 +13,18 @@ RequestStudentController.save = async (req, res, next) => {
...
+    const request = await RequestStudentService.findByStudentId(student_id);
```
Comment (ingesofteam) [inline app/controllers/RequestStudentController.js]: const request = await RequestStudentService.findByStudentId(studentId);

File: app/services/RequestStudentService.js [hunk 4]
```diff
@@ -11,3 +11,10 @@ RequestStudentService.create = async (request) => {
 
   return RequestStudentRepository.create(request);
 };
+
+// eslint-disable-next-line camelcase
+RequestStudentService.findByStudentId = (student_id) => {
```
Comment (ingesofteam) [inline app/services/RequestStudentService.js]: igual aca

File: app/repositories/RequestStudentRepository.js [hunk 5]
```diff
@@ -4,3 +4,6 @@ const DB = require('../utils/DB');
 RequestStudentRepository.create = requestStudent => DB('request_student').insert(requestStudent).returning('*');
 
 RequestStudentRepository.find = id => DB('request_student').select('*').where({ id }).first();
+
+// eslint-disable-next-line camelcase
+RequestStudentRepository.findByStudentId = student_id => DB('request_student').select('*').where({ student_id });
```
Comment (ingesofteam) [inline app/repositories/RequestStudentRepository.js]: studentId

Comment (ingesofteam) [reply app/repositories/RequestStudentRepository.js]: DB('request_student').select('*').where({ student_id: studentId });

**Pull request merged**